gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/io/IoTest.java

```diff
@@ -721,14 +732,18 @@ public static void assertCrewGraph(final Graph g1, final boolean lossyForId) {
     }
 
     public static void assertClassicGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId) {
-        assertToyGraph(g1, assertDouble, lossyForId, false);
+        assertToyGraph(g1, assertDouble, lossyForId, false, true);
+    }
+
+    public static void assertNoEdgeGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId) {
+        assertToyGraph(g1, assertDouble, lossyForId, false, false);
     }
 
     public static void assertModernGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId) {
-        assertToyGraph(g1, assertDouble, lossyForId, true);
+        assertToyGraph(g1, assertDouble, lossyForId, true, true);
     }
 
-    private static void assertToyGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId, final boolean assertSpecificLabel) {
+    private static void assertToyGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId, final boolean assertSpecificLabel, final boolean assertEdgeLabel) {
```

Comment:
That is basically what I do - I produce tinkerpop-no-edge-labels.xml. The assert function needs to know if it should check the particular labels values or just to assume that they have default values and check them.